packages/gatsby-cli/README.md

```diff
@@ -59,11 +59,11 @@ At the root of a Gatsby app run `gatsby build` to do a production build of a sit
 
 #### Options
 
-|            Option            | Description                                                                                                 | Default |
-| :--------------------------: | ----------------------------------------------------------------------------------------------------------- | :-----: |
-|       `--prefix-paths`       | Build site with link paths prefixed (set pathPrefix in your config)                                         | `false` |
-|        `--no-uglify`         | Build site without uglifying JS bundles (for debugging)                                                     | `false` |
-| `--open-tracing-config-file` | Tracer configuration file (open tracing compatible). See https://www.gatsbyjs.org/docs/performance-tracing/ |         |
+|            Option            | Description                                                                                                | Default |
+| :--------------------------: | ---------------------------------------------------------------------------------------------------------- | :-----: |
+|       `--prefix-paths`       | Build site with link paths prefixed (set pathPrefix in your config)                                        | `false` |
+|        `--no-uglify`         | Build site without uglifying JS bundles (for debugging)                                                    | `false` |
+| `--open-tracing-config-file` | Tracer configuration file (OpenTracing compatible). See https://www.gatsbyjs.org/docs/performance-tracing/ |         |
```

Comment:
```suggestion
| `--open-tracing-config-file` | Tracer configuration file (OpenTracing compatible). See  https://gatsby.dev/tracing |         |
```

Reply:
I didn't change on purpose (well actually reverted the change I did by mistake) - because this is rendered either on gatsby site (where we would lose SPA routing), or npm readme or github md viewer so link is not visible text and using full canonical path is ok.

Reply:
oh nice one!